Installing-Oppia-using-Docker.md

```diff
@@ -202,13 +210,14 @@ These are the `make` commands available for running tests:
 - `make run_tests.backend` - Runs the backend tests, verifies the functionality of backend logic and database interactions.  
 **Flags:**  
 	Usage - `make run_tests.backend PYTHON_ARGS="--flag_name"`
-	* `--test_target` - Specifies the dotted module name of the test(s) to run.
+	* `--test_targets` - Specifies the dotted module name of the test(s) to run.
 	* `--test_path` - Specifies the subdirectory path containing the test(s) to run.
 	* `--test_shard` - Specifies the name of the shard to run.
 	* `--generate_coverage_report` - Generates a coverage report.
 	* `--ignore_coverage` - Prevents tests from failing due to coverage issues.
 	* `--exclude_load_tests` - Excludes load tests from being run.
 	* `--verbose` - Displays the output of the tests being run.
+   * `--run_on_changed_files_in_branch` - Runs only the backend tests for files that have been altered in the current branch.
```

Comment:
Just noting that this doesn't seem to have the same indentation as the lines above (on "Files Changed"). I think it's due to tabs vs spaces, can you please change everything to spaces only? Thanks.

Reply:
![Screenshot 2024-08-27 230805](https://github.com/user-attachments/assets/3e4fe315-6562-48ab-88d9-f64596acb393)
![Screenshot 2024-08-27 230828](https://github.com/user-attachments/assets/5eb32880-9c78-46ce-aa5d-3891a43e8a8b)
In my editor I am using spaces and it has the same indent, when I edit file directly through Github and change it to the same indentation, weirdly it pushes the text to sub bullet point.

Reply:
@jnvtnguyen Can you check whether the previous lines (the ones you didn't edit) are using spaces or tabs (in your editor + in GitHub)?

Reply:
Nevermind, was able to fix it by using vim instead of VSCode.